charts/seed-bootstrap/charts/fluent-bit/templates/_fluent-bit-config.tpl

```diff
@@ -27,6 +27,7 @@
         Exclude_Path      *_garden_fluent-bit-*.log,*_garden_loki-*.log
         Parser            docker
         DB                /var/log/flb_kube.db
+        DB.sync           full
```

Comment:
> Note that full has a high I/O performance cost.

How will fluent bit perform with this change on a large seed cluster, for example with 100+ shoots?

Reply:
The current integration test runs with 50 containers. I assume a large seed cluster should contain 10 times more ( ~500). 
So we will run additional (customized) tests to validate.

Reply:
On `dev` landscape `aws` seed with 67 shoot there is no difference.

Reply:
@vlvasilev @JordanJordanov shall we then go ahead merging this PR or do you want to perform further tests?

Reply:
We will perform more tests.
Actually we will enable 1st on dev, staging & CANARY via [ landscape.yaml](https://github.wdf.sap.corp/kubernetes-dev/landscape-dev-garden/blob/master/landscape.yaml#L827-L870) and only then we will change the default value for all landscapes

Reply:
I see. Shall we then close this PR for now and reopen again when it's relevant to not keep it open for a long time?

Reply:
/close

@vlvasilev please re-open once you have validated this change from performance point of view.